doc/source/whatsnew.rst

```diff
@@ -18,6 +18,10 @@ What's New
 
 These are new features and improvements of note in each release.
 
+.. include:: whatsnew/v0.22.0.txt
+
```

Comment:
you need to rebase on master. these are already present

Reply:
I don't think they are: https://raw.githubusercontent.com/pandas-dev/pandas/master/doc/source/whatsnew.rst

Reply:
@TomAugspurger hmm thought we turned this on

Reply:
rebase; this is fixed